app/helpers/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Class Helper Contains helper functions
+ */
+export default class Helper {
+  /**
+       * Group Games by their winners
+       * @param {Array} gamesArray - All games played
+       * @param {String} winner - The game winner
+       * @returns {Object} - The grouped leaderboard
+       */
+  static groupByWinner(gamesArray, winner) {
+    const grouped = gamesArray.reduce((acc, obj) => {
```

Comment:
Abbreviations `acc, obj` are not good variable or parameter names. Please use something more descriptive in this case. Also avoid using dataTypes `obj`, `gamesArray ` as variable/parameters names

Reply:
Please fix this and other instances of this in this entire PR

Reply:
Fixed